src/main/java/hudson/plugins/ansicolor/AnsiAttributeElement.java

```diff
@@ -65,6 +65,11 @@ public int hashCode() {
         return result;
     }
 
+    @Override
+    public String toString() {
+        return "<" + name + " " + attributes + ">";
```

Comment:
Just for debugging, maybe better to switch to `AnsiAttributeElement name={}, attributes={}` so we don't emit valid HTML directly in logs or something.

Reply:
Done in a95ea81.